runner/local/simple.go

```diff
@@ -2,69 +2,136 @@ package local
 
 import (
 	"fmt"
+	"sync"
+	"time"
+
 	"github.com/scootdev/scoot/runner"
 	"github.com/scootdev/scoot/runner/execer"
-	"sync"
 )
 
 func NewSimpleRunner(exec execer.Execer) runner.Runner {
+	return NewMultiRunner(exec, 1)
+}
+
+func NewMultiRunner(exec execer.Execer, maxRunning int) runner.Runner {
```

Comment:
I usually push after finishing with comments... I still have to address the missing tests. 






On Sat, Jul 30, 2016 at 11:56 AM -0700, "Dan Bentley" <notifications@github.com> wrote:












In runner/local/simple.go:
>  )
>  
>  func NewSimpleRunner(exec execer.Execer) runner.Runner {
> +	return NewMultiRunner(exec, 1)
> +}
> +
> +func NewMultiRunner(exec execer.Execer, maxRunning int) runner.Runner {



Seems like it's still here?



—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.